README.md

```diff
@@ -1,51 +1,35 @@
+|Time to First Response | The duration from creation to the initial comment or review.|
+|Time to Close | The period from creation to closure.|
+|Time to Answer (Discussions Only) | The time from creation to an answer.|
+|Time in Label | The duration from label application to removal, requires LABELS_TO_MEASURE env variable.|
 
-*For pull requests, these metrics exclude the time the PR was in draft mode.
 
-*For Issue and pull requests, issue/pull request author's own comments and comments by bots are excluded.
+* For pull requests, these metrics exclude the time the PR was in draft mode.
+* For Issue and pull requests, issue/pull request author's own comments and comments by bots are excluded.
 
-This action was developed by the GitHub OSPO for our own use and developed in a way that we could open source it that it might be useful to you as well! If you want to know more about how we use it, reach out in an issue in this repository.
+This action, developed by GitHub OSPO for our internal use, is open-sourced for your potential benefit.
+Feel free to inquire about its usage by creating an issue in this repository.
 
-To find syntax for search queries, check out the [documentation on searching issues and pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests) or the [documentation on searching discussions](https://docs.github.com/en/search-github/searching-on-github/searching-discussions).
+To find syntax for search queries, check out the [documentation on searching issues and pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests)
+or the [documentation on searching discussions](https://docs.github.com/en/search-github/searching-on-github/searching-discussions).
 
 ## Example use cases
 
-- As a maintainer, I want to see metrics for issues and pull requests on the repository I maintain in order to ensure I am giving them the proper amount of attention.
-- As a first responder on a repository, I want to ensure that users are getting contact from me in a reasonable amount of time.
-- As an OSPO, I want to see how many open source repository requests are open/closed, and metrics for how long it takes to get through the open source process.
-- As a product development team, I want to see metrics around how long pull request reviews are taking, so that we can reflect on that data during retrospectives.
+| Metric | Description |
+|--------|-------------|
+|Maintainer Metrics| Monitor issue and pull request metrics for effective attention.|
+|Timely Response| Ensure prompt user contact as a first responder.|
+|OSPO Insights| Track open source request status and processing times.|
+|Review Efficiency| Analyze PR review durations for retrospectives.|
```

Comment:
I prefer to keep the original here. I feel it better lines up with software industry's use case language.

Reply:
I also thought this aligned with the use case. I've reverted the changes.

https://github.com/github/issue-metrics/pull/139/commits/1b6d343111bc79863010ae787e429f9f23d62163